app/views/admin/merchants/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1 align = "center">Merchants</h1>
+<p align="center"><%= link_to "New Merchant", "/merchants/new" %></p>
+<section class = "grid-container">
+  <% @merchants.each do |merchant|%>
+    <section class = "grid-item">
+      <h2><%=link_to merchant.name, "/merchants/#{merchant.id}"%></h2>
+      <% if merchant.active? %>
+        <p>Active</p>
+        <p><%= link_to "Disable #{merchant.name}", "/admin/merchants/#{merchant.id}?activate=false", method: :patch %></p>
+      <% else %>
+        <p>Disabled</p>
+        <p><%= link_to "Activate #{merchant.name}", "/admin/merchants/#{merchant.id}?activate=true", method: :patch %></p>
```

Comment:
I took the layout from the merchants index and added links to enable or disable depending on the status of each merchant